module/zfs/vdev_raidz_math_avx512f.c

```diff
@@ -194,6 +194,8 @@ typedef struct v {
 		    "vpternlogd $0x6c,%zmm29, %zmm26, %" VR0(r) "\n"	\
 		    "vpternlogd $0x6c,%zmm29, %zmm25, %" VR1(r));	\
 		break;							\
+		default:				  \
+			VERIFY(0);			  \
```

Comment:
nit: it might be the Github rendering but can you please make sure the trailing `\` characters are properly tab-aligned to the same column for all the modified blocks.